.github/workflows/ci.yml

```diff
@@ -22,23 +23,16 @@ jobs:
     - name: Setup Node.js
       uses: actions/setup-node@v1
       with:
-        node-version: '12.16.1'
```

Comment:
Also, it seems like this version of Node has a problem with our setup which causes the build to fail. Can you use `v12.4.0`?